blog/_posts/2021-11-08-new-tutorial.md

```diff
@@ -0,0 +1,41 @@
+---
+slug: "new-tutorial"
+date: 2021-11-08
+author: lian
+layout: blog
+title: "Welcome to Tilt: Revamped!"
+image: "/assets/images/new-tutorial/pexels-harry-cunningham-harrydigital-3405489.jpg"
+image_caption: "Brown and red windmills in front of blue sky. Photo by Harry Cunningham @harry.digital from <a href='https://www.pexels.com/photo/brown-and-red-wind-mill-3405489/'>Pexels</a>"
+description: "making old things new"
+tags:
+  - tutorial
+  - tilt
+  - devx
+---
+Hallo Tilters,
+
+Big announcement incoming: we have redesigned the Tilt experience! 🥳  
+Today we’re taking a look at our onboarding. Most of you who are reading this are probably already acquainted with Tilt— don’t worry, we’ve got something coming for you, too.
+
+Everyone knows: you can only make one first impression, better make it count! So with that in mind, we wanted to create an experience that is quick, easy and fun, even if you’re starting from zero. That goes for both yourself and the friends you want to introduce Tilt to!
+
+## Onboarding yourself
+
+So we went out, collected, considered and reflected on the feedback all of you provided (thanks again for that!) and overhauled the entire thing!  
+The new tutorial helps you to get Tilt up and running yourself. With just one command, you can quickly tour Tilt’s best features, and be inspired by all the possibilities to boost your own developer workflow.  
+All you need to get started, is Docker, Tilt and the sample source code. You’ll be able to play around with an actual multiservice application and see Tilt in action. [Try it out yourself](https://docs.tilt.dev/tutorial/index.html) and let us know what you think.
+
+If you’d rather learn how to write your first Tiltfile to start, the [old tutorial](https://docs.tilt.dev/tiltfile_authoring.html) is still up.
+
+![Screenshot of the new Tilt docs showing the Tutorial overview](/assets/images/new-tutorial/docs-tutorial.png)
+
+## Onboarding your friends
+
+Still reading? Excellent!  
+Next step: getting your colleagues and managers on board! You might have asked yourself before: How can I quickly show Tilt to my buddy, but on their machine?  
+To help you with that, we’ve created a self-contained demo. Just run `tilt demo` and Tilt will spin up an entire dev cluster for you with our sample application. From scratch!
```

Comment:
We do need Docker installed though, don't we? `From scratch!` makes it sound like you don't.